### PR TITLE
allow adapter to be required

### PR DESCRIPTION
**Pull request by fippo**
minor change from the previous PR: I don't think it makes sense to export trace

### CODE REVIEW

**Comment by fippo** [comment]
hey @googlebot, didn't you see I was already a contributor. But I've updated my CLA and will get you a corporate one even ;-)

File: adapter.js [hunk 1]
```diff
@@ -228,3 +228,16 @@ function requestUserMedia(constraints) {
...
+
+if (typeof module !== 'undefined') {
+  module.exports = {
+    RTCPeerConnection: RTCPeerConnection,
+    getUserMedia: getUserMedia,
+    attachMediaStream: attachMediaStream,
+    reattachMediaStream: reattachMediaStream,
+    webrtcDetectedBrowser: webrtcDetectedBrowser,
+    webrtcDetectedVersion: webrtcDetectedVersion,
+    requestUserMedia: requestUserMedia
```
Comment (juberti) [inline adapter.js]: i would prefer that requestUserMedia not be used as well, this should become an overload of getUserMedia

**Comment by juberti** [comment]
lgtm with one comment

**Comment by fippo** [comment]
Done. commented out so they're not accidentially added again.

**Pull request merged**